scripts/vtime_dist.bt

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bpftrace
+// Copyright (c) Meta Platforms, Inc. and affiliates.
+
+// This software may be used and distributed according to the terms of the
+// GNU General Public License version 2.
+
+
+rawtracepoint:sched_wakeup,
+rawtracepoint:sched_wakeup_new,
+{
+	$task = (struct task_struct *)arg0;
+
+	if ($1 > 0 && $task->tgid != $1) {
+		return;
+	}
+
+	if ($task->scx.dsq->id >= 0) {
+		@dsq_vtime[$task->scx.dsq->id] = hist($task->scx.dsq_vtime);
+	}
+}
+
+
+interval:s:1 {
+    $scx_ops = kaddr("scx_ops");
```

Comment:
When I run the script with 6.12-rc2 kernel (CachyOS), it gives the following error message. 

ERROR: Failed to compile: Failed to resolve kernel symbol: scx_ops

If I remove the three slides using scx_ops, it works fine.

Reply:
I'll make this optional then so if the script is ran with `./vtime_dist.bt 1` it will print the scheduler.